MAINTAINERS.md

```diff
@@ -0,0 +1,57 @@
+| Ishita Sequeira           | [ishitasequeira](https://github.com/ishitasequeira)     | Approver - CD                                  | [Red Hat](https://www.github.com/redhat/)            |
+| Ashutosh Singh            | [ashutosh16](https://github.com/ashutosh16)             | Approver(docs) - CD                            | [Intuit](https://www.github.com/intuit/)             |
+| Linghao Su                | [linghaoSu](https://github.com/linghaoSu)               | Reviewer - CD                                  | [DaoCloud](https://daocloud.io)                      |
+| Jesse Suen                | [jessesuen](https://github.com/jessesuen)               | Lead - CD                                      | [Akuity](https://akuity.io/)                         |
+| Yuan Tang                 | [terrytangyuan](https://github.com/terrytangyuan)       | Reviewer - CD                                  | [Red Hat](https://www.github.com/redhat/)            |
+| William Tam               | [wtam2018](https://github.com/wtam2018)                 | Reviewer - CD                                  | [Red Hat](https://www.github.com/redhat/)            |
+| Ryan Umstead              | [rumstead](https://github.com/rumstead)                 | Approver - CD                                  | [Black Rock](https://www.github.com/blackrock/)      |
+| Regina Voloshin           | [reggie-k](https://github.com/reggie-k)                 | Approver - CD                                  | [Codefresh](https://www.github.com/codefresh/)       |
+| Hong Wang                 | [wanghong230](https://github.com/wanghong230)           | Reviewer - CD                                  | [Akuity](https://akuity.io/)                         |
+| Jonathan West             | [jgwest](https://github.com/jgwest)                     | Approver - CD                                  | [Red Hat](https://www.github.com/redhat/)            |
+
+## Alumni
+
+| Alumni            | GitHub ID                                     | Project Roles        | Affiliation                              |
+|-------------------|-----------------------------------------------|----------------------|------------------------------------------|
+| Remington Breeze  | [rbreeze](https://github.com/rbreeze)         | Approver - CD        | [Akuity](https://akuity.io/)             |
+| Yi Cai            | [ciiay](https://github.com/ciiay)             | Reviewer - CD        | [Red Hat](https://www.github.com/redhat/)|
+| Saumeya Katyal    | [saumeya](https://github.com/saumeya)         | Reviewer - CD        | [Red Hat](https://www.github.com/redhat/)|
+| Regina Scott      | [reginapizza](https://github.com/reginapizza) | Reviewer - CD        | [Red Hat](https://www.github.com/redhat/)|
+| May Zhang         | [mayzhang2000](https://github.com/mayzhang2000)| Approver - CD        | [Intuit](https://www.github.com/intuit/) |
```

Comment:
We can probably remove this since the main purpose of the doc is to help LFX Insights understand who our current maintainers are.